lib/phx_tools_web/live/phx_tools_components.ex

```diff
@@ -92,7 +126,7 @@ defmodule PhxToolsWeb.PhxToolsComponents do
             <div class="md:flex">
```

Comment:
Do we still need this `os_landing_card` function? The other one seems to catch all the pages (we have only one now).